### PR TITLE
Update production companies

### PR DESCRIPTION
**Pull request by Mussuranae**
Replace productionCompanies by stakeholders

### CODE REVIEW

File: libs/movie/src/lib/movie/form/main/main.component.html [hunk 1]
```diff
@@ -139,8 +139,8 @@ <h3> Total Run Time </h3>
   </mat-form-field>
   <!-- PRODUCTION COMPANIES -->
   <h3> Production companies </h3>
-  <div formArrayName="productionCompanies"
-    *ngFor="let productionCompany of main.productionCompanies.controls; let i = index">
+  <div formArrayName="stakeholders"
+    *ngFor="let productionCompany of main.stakeholders.controls; let i = index">
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.component.html]: rename

File: libs/movie/src/lib/movie/form/main/main.component.html [hunk 2]
```diff
@@ -139,8 +139,8 @@ <h3> Total Run Time </h3>
...
     <div [formGroupName]="i" class="full-width">
       <mat-form-field appearance="outline" class="small-margin">
         <mat-label>Company name</mat-label>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.component.html]: change label

File: libs/movie/src/lib/movie/form/main/main.component.html [hunk 3]
```diff
@@ -154,7 +154,7 @@ <h3> Production companies </h3>
       </button>
     </div>
   </div>
-  <ng-container *ngIf="main.productionCompanies.controls.length === 0">
+  <ng-container *ngIf="main.stakeholders.controls.length === 0">
     <button mat-icon-button color="primary" (click)="main.addProductionCompany()">
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.component.html]: rename AddProductionCompany

File: libs/movie/src/lib/movie/form/main/main.form.ts [hunk 4]
```diff
@@ -36,20 +36,20 @@ function createDirectorFormControl(director?: Partial<Credit>) {
 
 type DirectorFormControl = ReturnType<typeof createDirectorFormControl>;
 
-export class ProductionCompagnyForm extends FormEntity<ProductionCompagnyControl> {
+export class StakeholdersForm extends FormEntity<StakeholdersControl> {
   constructor(compagny?: Partial<Credit>) {
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.form.ts]: rename compagny

Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.form.ts]: In `constructor(compagny?: Partial<Credit>) ` rename Credit by Stakeholder

File: libs/movie/src/lib/movie/form/main/main.form.ts [hunk 5]
```diff
@@ -36,20 +36,20 @@ function createDirectorFormControl(director?: Partial<Credit>) {
...
-    super(createProductionCompagnyControl(compagny))
+    super(createStakeholdersControl(compagny))
   }
 }
 
-function createProductionCompagnyControl(compagny?: Partial<Credit>) {
+function createStakeholdersControl(compagny?: Partial<Credit>) {
   const { displayName } = createCompany(compagny);
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.form.ts]: rename method createCompany and compagny

File: libs/movie/src/lib/movie/form/main/main.form.ts [hunk 6]
```diff
@@ -121,11 +121,11 @@ export class MovieMainForm extends FormEntity<MovieMainControl>{
   }
 
   public addProductionCompany(): void {
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.form.ts]: rename method

File: libs/movie/src/lib/movie/form/main/main.form.ts [hunk 7]
```diff
@@ -121,11 +121,11 @@ export class MovieMainForm extends FormEntity<MovieMainControl>{
...
-    const credit = new ProductionCompagnyForm();
-    this.productionCompanies.push(credit);
+    const credit = new StakeholdersForm();
+    this.stakeholders.push(credit);
   }
 
   public removeProductionCompany(i: number): void {
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/main.form.ts]: same

File: libs/movie/src/lib/movie/form/main/production-companies/production-companies.component.ts [hunk 8]
```diff
@@ -13,7 +12,7 @@ export class ProductionCompaniesComponent {
   @Input() form: FormList<Company>;
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/main/production-companies/production-companies.component.ts]: Use StakeHolder (even it stakeHolder & Company interface are equal)

File: apps/catalog/catalog/src/app/marketplace/movie/view/view.component.html [hunk 9]
```diff
@@ -88,7 +88,7 @@ <h1>{{movie.main.title.international}}</h1>
           [synopsis]="movie.main.shortSynopsis"
           [keywords]="movie.promotionalDescription.keywords"
           [releaseDate]="movie.AFMOriginCountryReleaseDate"
-          [productionCompanies]="movie.main.productionCompanies"
+          [productionCompanies]="movie.main.stakeholders"
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/marketplace/movie/view/view.component.html]: rename productionCompanies

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/credits/credits.component.html [hunk 10]
```diff
@@ -8,8 +8,7 @@ <h1 subtitle>Credits</h1>
       </mat-card-header>
       <mat-card-content>
         <movie-form-production-year [form]="productionYear"></movie-form-production-year>
-        <movie-form-production-companies [form]="productionCompanies"></movie-form-production-companies>
-        <!-- TODO#1478 co-production company -->
+        <movie-form-production-companies [form]="stakeholders"></movie-form-production-companies>
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/credits/credits.component.html]: rename component ` <movie-form-production-companies`

File: libs/movie/src/lib/movie/form/main/main.form.ts (hunk 5 above)
Comment (Mussuranae) [reply libs/movie/src/lib/movie/form/main/main.form.ts]: There is already a `createStakeholder` in the identity.ts file, but the fields won't match if I change to it

File: styles/themes/light.scss [hunk 11]
```diff
@@ -110,17 +110,17 @@ $bf-light-background: (
   app-bar: #FFF,
   status-bar: #FFF,
   background: #F4F4F4,
-  disabled-button: #EAEFF4,
+  disabled-button: #D1DBE6,
```
Comment (phpgeekfr) [inline styles/themes/light.scss]: Modification of this CSS file was intended ?

Comment (Mussuranae) [reply styles/themes/light.scss]: Yes Mathilde changed few color for disabled things, and I don't want to do a PR just for that so I added it to this PR, sorry :3

Comment (phpgeekfr) [reply styles/themes/light.scss]: No problem, I do it often also. Was just checking if it was a mistake or not

**Pull request merged**